compute/rn/release-information/release-notes-22-06-update2.adoc

```diff
@@ -46,11 +46,7 @@ The following table provides the release details:
 * Fixed an explicit deny error when sending a test alert to the AWS Security Hub integration. With this fix, the correct AWS product ARN for US and China regions are used.
 
 // GH#32310 PCSUP-10507
-* Fixed a bug in scanning images that mount on a directory when a parent directory is a symlink, causing many false binary detections.
-OR
-* Fixed a bug in scanning images on container defenders installed on CRI with CRI-O containerd that mount on a directory when a parent directory is a symlink, causing many false binary detections.
-OR
-* The image scans on container defenders installed on CRI with CRI-O containerd running from the host namespace are ignored to avoid false binary vulnerability detections.
+* Fixed an issue for defenders running on containerd/CRI-O nodes where the image had a mount point to the host filesystem and some parent directory of the mount point was a symlink, leading to host scanning.
```

Comment:
Need to add also this point: 
* Fixed an issue for defenders running on containerd/CRI-O nodes where during image scans for containers that changes to the host mount namespace, the host filesystem was scanned.

2 issues were fixed here